ci/libvcx.dockerfile

```diff
@@ -1,81 +1,76 @@
-FROM base:1.0.0 AS BASE
+FROM alpine:3.12 AS builder
 
-# Install Rust toolchain
-ARG RUST_VER=1.40.0
-RUN curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain ${RUST_VER}
-ENV PATH /home/indy/.cargo/bin:$PATH
+ARG UID=1000
+ARG GID=1000
```

Comment:
Let's make these ARGs global, since we reuse them (i.e. define them before the base image).